EIPS/eip-5851.md

```diff
@@ -0,0 +1,254 @@
+    },
+    "logic": "and",
+    "values":{"30000", "5"}
+}); 
+will correspond to the the functionality that admin needs to adjust the standard requirement for the identification SBT with tokenId = 1, based on the conditions described in the Requirements array struct details.
+**/
+    function changeStandardRequirement(uint256 SBTID, Requirement[] memory requirements) external returns (bool);
+
+    /// @notice function which uses the ZKProof protocol in order to validate the identity based on the given 
+    /// @dev it should only be called by the admin address.
+    /// @param SBTID is the Id of the SBT based identity certificate for which admin wants to define the Requirements.
+    /// @param certifying is the address that needs to be proven as the owner of the SBT defined by the tokenID.
+    /// @param `requirements` is the struct array of all the descriptions of condition metadata that is defined by the administrator. check metadata section for more information.
+    function certify(address certifying, uint256 SBTID) external returns (bool);
+
+    /// @notice function which uses the ZKProof protocol in order to validate the identity based on the given 
+    /// @dev it should only be called by the admin address.
+    /// @param SBTID is the Id of the SBT based identity certificate for which admin wants to define the Requirements.
+    /// @param certifying is the address that needs to be proven as the owner of the SBT defined by the tokenID.
+    /// @param `requirements` is the struct array of all the descriptions of condition metadata that is defined by the administrator. check metadata section for more information.
+    // eg: revoke(0xfoo,1): means that KYC admin revokes the SBT certificate number 1 for the address '0xfoo'.
+
+    function revoke(address certifying, uint256 SBTID) external returns (bool);
+
+}
+```
+
+### events
+
+```solidity
+
+/** 
+
+    interface IERC5851 {
+    * standardChanged
+    * @notice standardChanged MUST be triggered when requirements are changed by the admin. 
+    * @dev standardChanged MUST also be triggered for the creation of a new SBTID.
+    e.g : emit standardChanged(1,Requirement(Metadata('depositRequirement','number', 'defines the max deposited that user can have in denomination of USDC' ), "<=", "30000");
+    defines that holder of the identifier has been changed to the condition which allows the certificate holder to call the functions with modifier , only after the deposit in the address is not greater than 30000 USDC.
+    */
+    event standardChanged(uint256 SBTID, Requirement[] _requirement);
+    /** 
+    * certified
+    * @notice certified MUST be triggered when SBT certificate is given to the certifiying address. 
+    * eg: certified( 0xfoo,2); means that wallet holder address 0xfoo is certified to hold certificate issued with id 2 , and thus can satisfy all the confitions defined by the required interface.
+    */
+    event certified(address certifying, uint256 SBTID);
```

Comment:
Would prefer PascalCase for events i.e `Certified` to differentiate from function